soh/soh/UIWidgets.cpp

```diff
@@ -263,7 +263,9 @@ namespace UIWidgets {
 
     void EnhancementSliderInt(const char* text, const char* id, const char* cvarName, int min, int max, const char* format, int defaultValue, bool PlusMinusButton) {
         int val = CVar_GetS32(cvarName, defaultValue);
+        Spacer(0);
         ImGui::Text(text, val);
+        Spacer(0);
```

Comment:
The first spacer isn't neccesary because the spacing between UI elements is done outside of these functions.